util.py

```diff
@@ -23,15 +24,31 @@ def get_economics_status() -> Dict[str, Any]:
         status = {
             'latest_update': int(datetime.datetime.now().timestamp()),
             'backend': SERVER,
-            'total_supply': total_supply,
-            'circulating_supply': circulating_supply,
-            'mined_tokens': mined_tokens,
-            'released_premined_tokens': RELEASED_PREMINED_TOKENS
+            'total_supply':
+                get_decimals(total_supply, add_comma_separator) if decimals else total_supply,
+            'circulating_supply':
+                get_decimals(circulating_supply, add_comma_separator) if decimals else circulating_supply,
+            'mined_tokens':
+                get_decimals(mined_tokens, add_comma_separator) if decimals else mined_tokens,
+            'released_premined_tokens':
+                get_decimals(RELEASED_PREMINED_TOKENS, add_comma_separator) if decimals else RELEASED_PREMINED_TOKENS,
         }
 
     return status
 
 
+def get_decimals(value: int, add_comma_separator: bool) -> str:
+    assert sys.version_info >= (3, 6)
+    value /= 10**DECIMAL_PLACES
+    # This fixes the case where the decimal places are .00, so we must return with all zeros
+    string_formatter = '{:.{}f}'
+    if add_comma_separator:
```

Comment:
Why the comma separator? I mean, why are we returning a string instead of a number?

Reply:
The string is needed to add all decimal places when we have '.00'. The number would be 1234.0 instead of 1234.00, what could be confusing for some people not knowing the total number of decimal places.

Because of that, I thought it was easier to read the number with the comma separators, at least for the status API, which is the one I wanted to share in our economics page and is not used for coingecko.

Reply:
Ok. Thanks! I agree that it is way easier to read. Maybe we should just allow to enable/disable the comma separator in the querystring.